okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -1125,4 +1126,28 @@ public void redirectUsingNewConnectionEventSequence() throws IOException {
         "CallEnd");
     assertEquals(expectedEvents, listener.recordedEventTypes());
   }
+
+  @Ignore("Header added by interceptor is not seen by EventListener.")
+  @Test
+  public void requestModifiedByInterceptor() throws IOException {
+    server.enqueue(new MockResponse());
+
+    client
```

Comment:
You need to assign this to the client again.

Reply:
Good catch. Fixed.